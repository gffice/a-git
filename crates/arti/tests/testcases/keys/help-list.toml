bin.name = "arti"
args = "keys list --help"
status.code = 0
