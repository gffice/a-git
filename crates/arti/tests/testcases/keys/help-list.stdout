List keys and certificates

Usage: arti keys list [OPTIONS]

Options:
  -c, --config <FILE>                 Specify which config file(s) to read. Defaults to [File("[..]"), Dir("[..]")]
  -k, --keystore-id <KEYSTORE_ID>     Identifier of the keystore. If omitted, keys and certificates from all the keystores will be returned
  -o <KEY=VALUE>                      Override config file parameters, using TOML-like syntax.
  -l, --log-level <LEVEL>             Override the log level (usually one of 'trace', 'debug', 'info', 'warn', 'error').
      --disable-fs-permission-checks  Don't check permissions on the files we use.
  -h, --help                          Print help
